lib/audited/auditor.rb

```diff
@@ -138,10 +155,6 @@ def non_audited_columns
 
       protected
 
-      def non_audited_columns
```

Comment:
Not directly related to this change but this function is just a duplicate deceleration of the one above it.